api/src/main/java/org/infinispan/api/common/CacheWriteOperationOptions.java

```diff
@@ -0,0 +1,74 @@
+package org.infinispan.api.common;
+
+import java.time.Duration;
+import java.util.EnumSet;
+
+/**
+ * @since 14.0
+ **/
+public interface CacheWriteOperationOptions extends CacheOperationOptions {
```

Comment:
what about removing the word "Operation"?
`CacheOperationOptions` -> `CacheOptions`
`CacheWriteOperationOptions` -> `CacheWriteOptions`

Reply:
Agreed.